ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -170,6 +208,20 @@ function ExceptionRequestDetailsPage() {
 
     const vulnerabilityState = getVulnerabilityState(vulnerabilityException);
 
+    const searchFilter = {
+        CVE: relevantCVEs.join(','),
+        Image: getImageScopeSearchValue(scope),
+    };
+
+    if (isPlatformCveSplitEnabled) {
+        searchFilter['Platform component'] =
```

Comment:
For consistency, is this `'Platform Component'` elsewhere, like:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/WorkloadCvesPage.tsx#L42

Reply:
Oh good catch - the same thing led to a bug in a WIP branch I had so definitely worth fixing now.

I wonder what the level of effort and benefits would be to having all possible BE search options in a central location in the UI, so that the `SearchFilter` is safer as a `Record<SearchOption, string[]>` instead of a `Record<string, string[]>`.

Reply:
Yes, I agree. Benefit was worth cost for feature flag and permission resource strings.